bin/run-tests.js

```diff
@@ -81,7 +81,13 @@ run()
     process.exit(0); // eslint-disable-line n/no-process-exit
   })
   .catch(function (err) {
-    console.error(chalk.red(err.toString()));
+    console.error(chalk.red(`Error!`));
+
+    if ('passed' in err) {
```

Comment:
This change will print out:
```
{ passed: 40140, failed: 4, total: 40144, runtime: 47482 }
```
where as before it would print out `[object Object]`